ksql-api/src/main/java/io/confluent/ksql/api/server/ServerVerticle.java

```diff
@@ -85,123 +71,18 @@ public void stop(final Promise<Void> stopPromise) {
 
   private Router setupRouter() {
     final Router router = Router.router(vertx);
-    router.route(HttpMethod.POST, "/query-stream").handler(BodyHandler.create())
-        .handler(this::handleQueryStream);
-    router.route(HttpMethod.POST, "/inserts-stream").handler(this::handleInsertsStream);
+    router.route(HttpMethod.POST, "/query-stream")
+        .produces("application/vnd.ksqlapi.delimited.v1")
+        .produces("application/json")
+        .handler(BodyHandler.create())
```

Comment:
QQ: does the use of `BodyHandler` here mean the user can't provide a stream of query requests, as its trying to "gather the entire request body"?

Reply:
Yes, BodyHandler gathers up the entire request body as a single buffer.